src/SprykerEco/Yves/Easycredit/Theme/default/components/molecules/easycredit-badge/easycredit-badge.twig

```diff
@@ -0,0 +1,27 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'easycredit-badge',
+    tag: 'easycredit-badge'
+} %}
+
+{% define data = {
+    title: null,
+    id: required
+} %}
+
+{% define attributes = {
+    'rk-options': required
+} %}
+
+{% block body %}
+    {% include molecule('script-loader') with {
+        class: 'script-loader-rk-plugin',
+        attributes: {
+            src: 'https://ratenkauf.easycredit.de/ratenkauf/js/ratenrechner/v1/ratenrechner.js'
+        }
+    } only %}
+    <br>
```

Comment:
Why do you need `<br>` here? If you need some spacing just use `spacing` util class.